crates/ruma-events/Cargo.toml

```
@@ -51,6 +51,7 @@ unstable-msc4274 = []
 unstable-msc4278 = []
 unstable-msc4319 = []
 unstable-msc4310 = []
+unstable-msc4334 = ["language-tags"]
```

Otherwise it adds a `language-tags` feature too:

```suggestion
unstable-msc4334 = ["dep:language-tags"]
```